MySQL.Data/src/MySQLActivitySource.cs

```diff
@@ -81,7 +81,7 @@ internal static Activity CommandStart(MySqlCommand command)
       activity?.SetTag("db.system", "mysql");
       activity?.SetTag("db.name", command.Connection.Database);
       activity?.SetTag("db.user", command.Connection.Settings.UserID);
-      activity?.SetTag("db.statement", command.OriginalCommandText);
```

Comment:
`OriginalCommandText` was a bit of a mystery to me - it may have made sense when the code looked different, but I don't understand when the sql query at time of call isn't good. I analysed the code paths in the PR description and they all seemed very understandable.

Is there an edge case I'm missing?